nestkernel/nestmodule.cpp

```diff
@@ -1623,6 +1623,9 @@ NestModule::GetStructuralPlasticityStatus_DFunction::execute( SLIInterpreter* i
 /**
  * Enable Structural Plasticity within the simulation. This means, allowing
  * dynamic rewiring of the network based on mean electrical activity.
```

Comment:
`This allows dynamic rewiring of the network based on mean electrical activity.`

Reply:
Thanks for your suggestion @sarakonradi